includes/admin/OrderSettings.php

```diff
@@ -611,6 +612,32 @@ public function getExtraOptions(): array
         return $this->extraOptions;
     }
 
+    /**
+     * Returns the weight of the order plus the empty parcel weight.
+     *
+     * @return int
+     */
+    public function getTotalWeight(): int
```

Comment:
Dit is wel een uitgelezen kans om de misleidende naam 'getTotalWeight' te refactoren. Deze geeft namelijk het gewicht PER PAKKETJE terug, NIET van de hele order. Ik stel me voor dat je een getTotalWeight hebt, en een getColloWeight ofzo die daar gebruik van maakt en de (juiste) verpakking erbij telt.